bin/ecs_rails_console

```diff
@@ -7,41 +7,53 @@ require 'ecs_rails_console'
 $LOAD_PATH.push File.expand_path('../lib', __dir__)
 
 options = {
-  environment: 'production'
+  environment: 'production',
+  command: 'bin/rails console'
 }
 
-OptionParser.new do |opts|
-  opts.banner = 'Usage: ecs_rails_console [options]'
+opts = OptionParser.new do |o|
+  o.banner = 'Usage: ecs_rails_console [options]'
 
-  opts.on('-g', '--generate-config', 'Generate config file') do
-    template    = File.expand_path('../lib/ecs_rails_console/templates/ecs_rails_console.yml',
-                                   __dir__)
+  o.on('-g', '--generate-config', 'Generate config file') do
+    template = File.expand_path('../config/ecs_rails_console.yml', __dir__)
     config_file = 'config/ecs_rails_console.yml'
 
-    FileUtils.mkdir_p(File.dirname(config_file))
-    FileUtils.cp(template, "#{Dir.pwd}/#{config_file}")
-    puts "File generated: #{config_file}"
+    if File.exist?(config_file)
+      puts 'Configuration file already exists and will be kept untouched.'
```

Comment:
There should probably be an option to over-write this rather than having to delete it manually. `--force-generate-config`?

Reply:
I don't know if this option would bring too much benefit and will add more logic.

Reply:
I want to turn this difficult to do exactly to prevent accidents. In order to reset your config file, you need to deliberately run 2 wrong commands in a row: remove the file then generate it again.

Reply:
That makes sense. Assuming you're not changing the file very often of course